### PR TITLE
Add nameserver 1.1.1.1 to address timeout issues

### PR DESCRIPTION
**Pull request by Shilpa-Gokul**

### CODE REVIEW

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-ppc64le-powervs

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-ppc64le-powervs

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-ppc64le-powervs

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-ppc64le-powervs

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-ppc64le-powervs

File: ci-operator/step-registry/openshift/e2e/libvirt/test/openshift-e2e-libvirt-test-ref.yaml [hunk 1]
```diff
@@ -58,6 +58,7 @@ ref:
       memory: 8Gi
   dnsConfig:
     nameservers:
+    - 1.1.1.1
```
Comment (prb112) [inline ci-operator/step-registry/openshift/e2e/libvirt/test/openshift-e2e-libvirt-test-ref.yaml:61]: Did this work for the load balancer monitor test?

Comment (Shilpa-Gokul) [reply ci-operator/step-registry/openshift/e2e/libvirt/test/openshift-e2e-libvirt-test-ref.yaml:61]: > Did this work for the load balancer monitor test?

In the CI runs, it passes sometimes so not completely sure if this fix resolves the issue or is actually passing. But the rehearsal jobs were all fine and the issue was not observed in the PR rehearsals

Comment (prb112) [reply ci-operator/step-registry/openshift/e2e/libvirt/test/openshift-e2e-libvirt-test-ref.yaml:61]: Super helpful Shilpa. Thank you

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-ppc64le-powervs

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-ppc64le-powervs

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-ovn-ppc64le-powervs

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-ppc64le-powervs

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-ovn-ppc64le-powervs

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-ovn-ppc64le-powervs

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-ppc64le-powervs

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-ppc64le-powervs

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-ovn-ppc64le-powervs

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-ppc64le-powervs

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-ppc64le-powervs

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-ppc64le-powervs-per

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-ppc64le-powervs-per

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-ppc64le-powervs

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-ppc64le-powervs-per

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-ppc64le-powervs

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-ppc64le-powervs

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-ppc64le-powervs-original

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-ppc64le-powervs-original

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-ppc64le-powervs-original

File: ci-operator/step-registry/openshift/e2e/powervs/ipi/openshift-e2e-powervs-ipi-workflow.yaml [hunk 2]
```diff
@@ -7,6 +7,14 @@ workflow:
...
     post:
     - chain: ipi-powervs-post
+    dnsConfig:
+      nameservers:
+        - 9.9.9.10
+        - 9.9.9.9
+        - 1.1.1.1
+        - 162.159.42.58
+        - 162.159.34.79
+        - 162.159.42.2
```
Comment (prb112) [inline ci-operator/step-registry/openshift/e2e/powervs/ipi/openshift-e2e-powervs-ipi-workflow.yaml:17]:
```suggestion
        - 9.9.9.9
        - 162.159.42.58
        - 162.159.34.79
```

Comment (prb112) [reply ci-operator/step-registry/openshift/e2e/powervs/ipi/openshift-e2e-powervs-ipi-workflow.yaml:17]: Discussed the above as an approach with Shilpa / Mike

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-ppc64le-powervs-original

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-ppc64le-powervs-original

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-ppc64le-powervs-original

**Comment by Shilpa-Gokul** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-ppc64le-powervs-original

**Comment by Shilpa-Gokul** [comment]
@mjturek could you please take a look?

**Pull request closed without merging**